package.json

```diff
@@ -28,7 +28,7 @@
     "dev:ssr": "run-p serve:ssr watch:*",
     "lint": "eslint .",
     "fixlint": "eslint . --fix",
-    "test": "mocha ./test/**/*.test.js",
+    "test": "npm run build && mocha ./test/**/*.test.js",
```

Comment:
I think it will still test against the source code instead of the build code ?

cause test files are taking modules from `src/*` directories, right ?

let me know if I missed anything

Reply:
I guess if this is so, we can have a separate tests, script for post build testing in a separate PR ?

Reply:
actually we don't need this change because the GitHub Action (`unit.yml`) already runs the production build.

Inside of `docsify.test.js`, it loads the production build file (`lib/docsify.js`).